product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx

```diff
@@ -457,6 +458,138 @@ witnessGroups:
+      - cidr: 9.9.9.9/28                                              # <string: CIDR of allowed source IP range>
+        description: Allow traffic from App A                         # <string: The description of this allowed ip range>
+      - cidr: 10.10.10.10/27                                          # <string: CIDR of allowed source IP range>
+        description: Allow traffic from App B                         # <string: The description of this allowed ip range>
+    pgConfigMap:                                                      # <Object: Postgres configuration>
+      application_name: test_app                                      # <string: set the database "application_name" property to "test_app">
+      array_nulls: true                                               # <bool: set the database "array_nulls" property to True>
+    backupRetentionPeriod: 30d                                        # <string: Retention period must be between 1-180 days or 1-25 weeks or 1-6 months. Using strings like "7d" or "2w" or "3m" to specify days, weeks and months respectively.>
+#    cspSubscriptionIds:                                              # <list:  cloud provider subscription/account ID, required when cluster is BigAnimal deployment.>
+#      - 123123123                                                    # <string>
+#      - 456456456                                                    # <string>
+#    serviceAccountIds:                                               # <list: A Google Cloud Service Account is used for logs. If you leave this blank, then you will be unable to access log details for this cluster. Required when cluster is BigAnimal deployment>
+#      - service-account-1234567b@development-data-123456.iam.gserviceaccount.com# <string>
+#      - service-account-1234567b@development-data-123456.iam.gserviceaccount.com# <string>
+
+witnessGroups:                                                        # <list: pgd witness groups
+  - provider: azure                                                   # <string:  provider id>
+    region: australiaeast                                                   # <string:  provider region id>
+    customMaintenanceWindow:                                          # <Object:  Schedule a custom maintenance window for witness group>
+      maintenanceStartTime: 18:00                                     # <string:  Set a start time in 24 hour format in UTC (e.g. 15:00) of day to initiate>
+      maintenanceStartDay: monday                                     # <string:  Select desired day (e.g. monday) from a week for maintenance activities>
+```
+
+#### Google Cloud example
+
+```
+clusterName: biganimal-gcp-cluster                                    # <string:  cluster name>
+password: password@12334                                                # <string:  cluster password (must be at least 12 characters)>
```

Comment:
lets keep it same across all examples i.e copy from `Azure example`

Reply:
Applied suggested edit